Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py

```diff
@@ -1760,6 +1810,21 @@ def get_alert_attachments_command(
     )
 
 
+def get_compromised_credentials_command(
+    client: ZFClient,
+    args: dict[str, Any]
+) -> CommandResults:
+    alert_id = args.get("alert_id", "")
```

Comment:
```suggestion
    alert_id = args["alert_id"]
```
This is required argument